packages/blade/src/components/Input/BaseInput/StyledBaseInput.native.tsx

```diff
@@ -1,5 +1,7 @@
-import type { ReactElement } from 'react';
+/* eslint-disable react/display-name */
```

Comment:
Let's actually add the display name. Will be easier to debug later.

Reply:
fixed in 717f7a1da44ddbaf76edc37e72d60e182d95576b